raiden/ui/cli.py

```diff
@@ -205,16 +211,15 @@ def app(address,
 
         address = addresses[idx]
 
-    password = os.environ.get('ETH_PASSWORD')
-    # if environ has password, try it
-    # avoid using click method to avoid password in command line
+    password = None
+    if password_file:
+        password = password_file.read().splitlines()[0]
```

Comment:
According to [documentation](http://click.pocoo.org/5/api/#click.File), it is when the context tears down, usually when command finishes executing. Tried using click's internal tool for small file open, to get a good error message when file doesn't exist or can't be opened and auto encoding handling, but I could use [click.Path](http://click.pocoo.org/5/api/#click.Path) and open the file by myself, if this file being open is an issue.